scripts/travis/05-update-2-3.sh

```diff
@@ -12,6 +12,7 @@ composer config repositories.thunder path ${THUNDER_DIST_DIR}
 composer remove burdamagazinorg/thunder --no-update
 composer require "thunder/thunder-distribution:*" --no-update
 
+composer update || true
```

Comment:
Does this mean, that people updating T2 => T3 also have to do this? This is very awkward :-(
Can we find a better solution here?

Reply:
@dbosen As I remember, the reason for 2nd composer update is that patch is still applied on the 1st run. On the 2nd run, it's removed properly. But, I'm not 100% sure.

What does it mean -> when Thunder 2 without patched paragraphs is released, we won't need 2nd composer update.